README.md

```diff
@@ -76,8 +76,8 @@ For complete documentation on installing the New Relic Ruby agent, see the follo
 
 ### Recording Deploys
 
-The Ruby Agent supports recording deployments in New Relic via a command line
-tool or Capistrano recipes. For more information on these features see
+The Ruby agent supports recording deployments in New Relic via a command line
+tool or Capistrano recipes. For more information on these features, see
```

Comment:
I wonder if you could work in something about our current bug about recording deploys using newly created API keys? It feels misleading as-is. 

Here's the issue I'm referring to: https://github.com/newrelic/newrelic-ruby-agent/issues/715

Reply:
Happy to add a new line referencing the bug, especially since it looks like a fix may take time. Not sure if this line covers the issue, but something like?: 

*There is a known issue that prevents newly generated NewRelic API keys from recoding deploys. See the latest on [this issue](https://github.com/newrelic/newrelic-ruby-agent/issues/715).

Reply:
I like it! Thanks!

Reply:
I'm happy to have the PR merged as is and address things with a follow-up, but I think we should do the following 2 things for the known issue warning:

1) I think we should add an asterisk to the word "supports" (so "supports*") to reference the footnote.
2) I think we can just have the footnote's "known issue" text be what gets linked, like so:

```markdown
There is a [known issue](https://github.com/newrelic/newrelic-ruby-agent/issues/715) that prevents newly generated New Relic API keys from recoding deploys.
````

Reply:
Thanks @fallwith. I'm adding your suggestion and tagging you for review/approval!